certsubmitter/certsubmitter.go

```diff
@@ -124,6 +124,13 @@ func checkSCT(sct *ct.SignedCertificateTimestamp, chain []*x509.Certificate, sv
 		errs = append(errs, &errors.SignatureVerificationError{Err: err})
 	}
 
+	// Verify SCT timestamp.
+	sctTimestamp := ct.TimestampToTime(sct.Timestamp)
+	// Consider only gaps bigger than 1 second
```

Comment:
nit: How about "Consider gaps bigger than 1 second as errors." Or the inverse "We accept timestamps up to 1 second after reception time by XXX." - ideally specifying who XXX that received the data is.

Would sctTimestamp.Sub(receivedAt).Milliseconds() < 1000 be a bit more readable?

Reply:
Adjusted.